frontend/src/audio.html

```diff
@@ -1,189 +1,27 @@
 ---
 permalink: "/audios/{{ audio.fileId | slug }}/"
 layout: base
-jsFile: audio.js
+jsFile: common-player.js
 description: Audio recording with all the details, including topics discussed
 ---
 <article class="mx-auto mt-10 flex max-w-screen-md flex-col px-2 sm:px-4">
   <div class="flex justify-between space-x-1 align-baseline">
     <!-- Title -->
     <h1 class="text-xl font-bold">{{ audio.title }}</h1>
-    <!-- Duration -->
-    <span class="inline-flex items-start">
-      <svg
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 24 24"
-        class="mx-1 my-2 w-4 stroke-current"
-        stroke-width="0.2"
-      >
-        <path
-          fill="currentColor"
-          d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8s8 3.58 8 8s-3.58 8-8 8zm-.22-13h-.06c-.4 0-.72.32-.72.72v4.72c0 .35.18.68.49.86l4.15 2.49c.34.2.78.1.98-.24a.71.71 0 0 0-.25-.99l-3.87-2.3V7.72c0-.4-.32-.72-.72-.72z"
-        />
-      </svg>
-      <p class="pt-1">{{ audio.duration | duration }}</p>
-    </span>
+    <div class="inline-flex content-end items-center">
+      {% render 'partials/duration.html' with audio.duration as duration -%}
+    </div>
```

Comment:
This surrounding `div` is an inherent part of the duration block. Why not to move it inside the partial?

Reply:
done